.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,15 @@
-#### What problem does the pull request solve?
+### What problem does this pull request solve?
 
-Trello card:
+Trello card: <!-- link -->
 
-#### Checklist
+<!-- Add some description here about what the PR is about, even if you have a Trello card to link to -->
```

Comment:
I'm being picky but I'm not a fan of things I need to delete. That said this is just a preference so happy to approve
```suggestion
```

Reply:
I don't think you need to delete this, as it is a HTML comment it will not appear in the GitHub UI.

Reply:
OHHHHHHHHHH my bad 🤦